src/connectors/answers/__tests__/connectAnswers-test.ts

```diff
@@ -0,0 +1,471 @@
+import algoliasearchHelper, { SearchResults } from 'algoliasearch-helper';
+import { createInstantSearch } from '../../../../test/mock/createInstantSearch';
+import { createSearchClient } from '../../../../test/mock/createSearchClient';
+import {
+  createInitOptions,
+  createRenderOptions,
+} from '../../../../test/mock/createWidget';
+import { createSingleSearchResponse } from '../../../../test/mock/createAPIResponse';
+import connectAnswers from '../connectAnswers';
+
+const defaultRenderDebounceTime = 10;
+const defaultSearchDebounceTime = 10;
+
+// 10(render debounce) + 10(search debounce) + 10(just to make sure)
+const wait = (time = 30) => {
+  return new Promise(resolve => {
+    setTimeout(resolve, time);
+  });
+};
```

Comment:
We could move this function to `test/utils` as it's now used/declared in multiple files (`createMetadataMiddleware.ts`, `history.test.ts`, and this one)

Reply:
Good point. Thanks for finding out those :)  https://github.com/algolia/instantsearch.js/pull/4581/commits/ea8a2f1f027ded9611bfd4ce7890787da7b86280